model/test_log.go

```diff
@@ -93,7 +96,16 @@ func (self *TestLog) Validate() error {
 // URL returns the path to access the log based on its current fields.
 // Does not error if fields are not set.
 func (self *TestLog) URL() string {
-	return fmt.Sprintf("/test_log/%v/%v/%v",
+	settings, err := evergreen.GetConfig()
+	if err != nil {
+		grip.Debug(message.WrapError(err, message.Fields{
+			"message":   "error getting config",
+			"operation": "set test url",
+			"task":      self.Task,
```

Comment:
You should either return or otherwise handle the case where settings is nil. Otherwise 108 will panic.

Reply:
right...obvious mistake sorry about that.